app/src/scripts/game/components/game.jsx

```diff
@@ -21,6 +21,10 @@ class Game extends Component {
 
   componentDidMount() {
     this.props.stopWalkthrough();
+    console.log(this.props.getOnboarding());
```

Comment:
This is not the right way to use your function and state.

--> getOnboarding() will fetch into chrome's google storage asynchronously, and then update your redux state (i.e. this.props...) once it the promise is completed.
--> getOnboarding() does NOT return any values. It's really important. It just updates the states. The way you get values should ALWAYS be via this.props..., not any functions.
--> You need to use `this.props.isOnboarded` to know whether the user is onboarded or not.
 
The value of `this.props.isOnboarded`  can be:
- `undefined`:  when the component is mounted, `this.props.isOnboarded` doesn't have any value. That means that the promise hasn't been returned yet.
- `false`: this means that the value has been returned from chrome storage, but is `false`, meaning that first walkthrough hasn't been done.
- `true`: that'll be the case most of the time. It means that the value has been returned from chrome storage and that the user has been onboarded.

```suggestion
    this.props.getOnboarding();
```